tutorials/compute-managed-instance-groups-dashboard/index.md

```diff
@@ -0,0 +1,182 @@
+the dashboard locally:
+
+1.  `git clone https://github.com/GoogleCloudPlatform/community`
+
+2.  `cd community/tutorials/compute-managed-instance-groups-dashboard/webapp/`
+
+3.  Edit `gapi.js` file: replace *clientId* in line 44 with your Client ID
+
+4.  From `webapp/` run: `python -m SimpleHTTPServer 8080`
+
+Now go to [http://localhost:8080](http://localhost:8080)
+in your browser, and you'll be greeted with a window which allows you to
+log in to your Google account. Choose the account which has the access
+to your GCP project. In the next window, Google will ask for your
+permission to allow the Dashboard to view and manage your data across
+GCP services.
+
+Once you grant the necessary permissions, you'll see an error stating
+that Google Cloud Resource Manager API has not been used in your project
+before. Follow the link from the error message to enable it. Give it a
+few minutes to propagate, go back to your Simple MIG Dashboard and
+refresh the page. Your Dashboard is ready to use.
+
+Overview of the dashboard
+-------------------------
+
+It's time to learn how to use the dashboard to monitor the state of your
+MIG:
+
+**Step 1.** Choose the name of the project and hit load.
+
+![Picking project](media/step1.png)
+
+**Step 2.** From the list of Managed Instance Groups in this project select
+the one you want to monitor.
+
+![Picking instance group](media/step2.png)
+
+**Step 3.** You are now monitoring your MIG.
+
+![MIG Dashboard main view](media/step3.png)
+
+1.  **Timespan** adjusts horizontal axis of *Instance status* chart.
+
+2.  **Group by zone** is visible only if your MIG is regional. When selected, it will change the order of the machines in the chart to display all machines in one zone together.
+
+3.  **Instance status** tab contains a chart that visualises how instances in your MIG were changing their state over time. In the example from the screenshot above:
+
+    -   Top three instances are using *regional-instance-7* instance template and have been stable for at least 3 minutes.
+
+    -   Three instances in the middle were running *regional-instance-7* instance template, but they are now being deleted for about 2 minutes and 30 seconds.
+
+    -   Bottom three instances were created around 2 minutes and 40 seconds ago from *regional-instance-1* instance template. They were in *CREATING* state for about 10 seconds, and then became stable.
+
+4.  **Summary** displays cumulative information about the status of the machines in the MIG. It allows you to check how many instances there are in each state. Together with *Group by zone*, the Summary tab allows you to see what is happening with your machines in each of the zones.
+
+5.  **Instance health** tab is visible only if your MIG is connected to a Backend Service and has a Health Check configured. If this is the case, each of your machines is reporting its health periodically. The chart displays the changes of health in time for each of the machines.
```

Comment:
change: "If this is the case, each of your machines is reporting its health periodically."
to
"In this case, each machine reports its health periodically.